onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1982,22 +1982,25 @@ void RegisterContribSchemas() {
             AttributeProto::INT, static_cast<int64_t>(ONNX_NAMESPACE::TensorProto_DataType_FLOAT))
       .AllowUncheckedAttributes()
       .Input(0, "X", "Input data tensor from the previous layer.", "T")
-      .Input(1, "Scale", "Scale tensor.", "T")
-      .Input(2, "B", "Bias tensor.", "T", OpSchema::Optional)
-      .Output(0, "Y", "Output data tensor.", "T")
+      .Input(1, "Scale", "Scale tensor.", "V")
+      .Input(2, "B", "Bias tensor.", "V", OpSchema::Optional)
+      .Output(0, "Y", "Output data tensor.", "V")
       .Output(1, "Mean", "Saved mean used during training to speed up gradient computation", "U", OpSchema::Optional)
       .Output(2, "InvStdDev", "Saved inverse standard deviation used during training to speed up gradient computation.", "U", OpSchema::Optional)
       .TypeConstraint(
           "T",
           {"tensor(float16)", "tensor(float)", "tensor(double)", "tensor(bfloat16)"},
-          "Constrain input types and output Y type to float tensors.")
+          "Constrain input X type to float tensors.")
       .TypeConstraint(
           "U",
```

Comment:
Not related to your change. 'U' did not allow double, how could register the kernel like this?  

      class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 1, double_double_double, LayerNormalization);

Reply:
Right. Should double instead of bfloat16.